src/runtime/providers/cloudimage.ts

```diff
@@ -21,17 +21,29 @@ const operationsGenerator = createOperationsGenerator({
   formatter: (key, value) => `${key}=${value}`
 })
 
+const isDev = process.env.NODE_ENV === 'development'
+
 // https://docs.cloudimage.io/go/cloudimage-documentation-v7/en/introduction
 export const getImage: ProviderGetImage = (src, {
   modifiers = {},
   baseURL = '',
-  token = 'demo',
+  token = '',
+  apiVersion = '',
   cdnURL = ''
 } = {}) => {
   const operations = operationsGenerator(modifiers)
   if (!cdnURL) {
-    cdnURL = `https://${token}.cloudimg.io/v7`
+    cdnURL = `https://${token}.cloudimg.io/${apiVersion}`
   }
+
+  if (isDev) {
```

Comment:
This makes it look like this provider will not work in development mode? Should we instead check `isDev && <some other condition>`?

Reply:
Oh man, I shouldn't do this late night. 
I've added proper condition now.